maxim/logger/livekit/llm.py

```
@@ -0,0 +1,97 @@
+    """Post-hook for LLM methods"""
+    try:
+        scribe().debug(
+            f"[Internal][{self.__class__.__name__}] {hook_name} completed; result_type={type(result).__name__}"
+        )
+    except Exception as e:
+        scribe().warning(
+            f"[{self.__class__.__name__}] {hook_name} post-hook failed; error={e!s}\n{traceback.format_exc()}"
+        )
+
+
+def instrument_llm_init(orig, name, class_name):
```

Ruff (0.12.2)

Missing return type annotation for public function `instrument_llm_init`

(ANN201)